scraper/load.py

```diff
@@ -0,0 +1,526 @@
+
+        try:
+            file = {
+                "image": open(f"media/{scrapped_id}_1.jpg", "rb")
+            }
+        except FileNotFoundError:
+            return
+        response = requests.post(
+            api_url+f"images/products/{product_id}",
+            files=file,
+            auth=(self.api_key, '')
+        )
+
+        if response.status_code == 200:
+            print(f"Image uploaded successfully: {scrapped_id}_1")
+        else:
+            print("Failed to upload image:", response.status_code, response.text)
+
+    def apply_combination(self, product_dict, attrib_name, product_id):
+        prices = product_dict["prices"]
+        indeks = product_dict["index"]
+        if attrib_name == "Waga":
+            weights = prices["weights"]
+            for index, weight in enumerate(weights):
+                price = round(prices["netto"][index] - self.base_weight_price, 2)
+                print("priiiiice", price)
+                #<ean13><![CDATA[1234567890123]]></ean13>
+                #<mpn><![CDATA[123456]]></mpn>
+                combination_xml = f"""<?xml version="1.0" encoding="UTF-8"?>
+                <prestashop xmlns:xlink="http://www.w3.org/1999/xlink">
+                    <combination>
+                        <id_product><![CDATA[{product_id}]]></id_product>
+                        <reference><![CDATA[{indeks[8:]}]]></reference>
+                        <supplier_reference><![CDATA[mfr_1]]></supplier_reference>
+                        <price><![CDATA[{price}]]></price>
+                        <minimal_quantity><![CDATA[1]]></minimal_quantity>
+                        <associations>
+                            <product_option_values nodeType="product_option_value" api="product_option_values">
+                                <product_option_value>
+                                    <id><![CDATA[{self.atribbs_map["Waga"][weight]["id"]}]]></id>
+                                </product_option_value>
+                            </product_option_values>
+                        </associations>
+                    </combination>
+                </prestashop>
+                """
+
+                response = self.make_request("POST", "combinations", combination_xml)
+
+                if response.status_code == 201:
+                    response_xml = ET.fromstring(response.text)
+                    atrib_id = response_xml.find('.//id').text
+
+                    print(f"Kombinacje atrybutu {attrib_name} dodano do produktu {product_id}")
+                else:
+                    print("Failed wartosc atrybutu:", response.status_code, response.text)
+
+        elif attrib_name == "Długość":
+            variants = prices["variants"]
+
+            for dlugosc, dowijki in variants.items():
+                for dowijka, ceny in dowijki.items():
+                    price = round(ceny["netto"] - self.base_weight_price, 2)
+                    print("priiiiice", price)
```

Comment:
We are serious organization 🗿
```suggestion
                    print(f"Price Impact: {price} PLN")
```